src/components/Areas.tsx

```diff
@@ -115,11 +115,11 @@ const Areas = () => {
   return (
     <>
       <Helmet>
-        <title>Areas | {meta.title}</title>
+        <title>Areas</title>
```

Comment:
Strictly speaking, we're not setting the title until we have the data. We could fix this with something like this:

```ts
const Areas = () => {
  // snip
}

const TitledAreas = () => (
  <>
    <Helmet>
        <title>Areas</title>
    </Helmet>
    <Areas />
  </>
);
```

However, that's a bit extreme -- I'll save that for a future PR. I think if/when we cared about the experience _that_ much, we should spend the time looking into server-side rendering instead.